linux/keyman-config/keyman_config/kvk2ldml.py

```diff
@@ -13,17 +11,18 @@
 # Magic   4 bytes 'KVKF'
 # Version 4 bytes 0x600
 # Flags   1 byte bitmask: [102key?, DisplayUnderlying?, UseUnderlying?, AltGr?]
-kvkk102key =              b'\x01'
-kvkkDisplayUnderlying =   b'\x02'
-kvkkUseUnderlying =       b'\x04'
-kvkkAltGr =               b'\x08'
+kvkk102key =              b'\x01'  # noqa: E222 - multiple spaces after operator
```

Comment:
The comment here pretty much undoes the good of aligning the variables. I don't really want to encourage this kinda of fighting-the-linter pattern 😢 because it leads to confusion from future maintainers. Just remove the multiple spaces.

Reply:
Done in #3275